src/domain/enrolments/filterSummary/FilterTag.tsx

```diff
@@ -0,0 +1,38 @@
+import { Tag } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { EnrolmentFilterType } from '../types';
+
+export interface FilterTagProps {
+  onDelete: (options: { type: EnrolmentFilterType; value: string }) => void;
+  text: string;
+  type: EnrolmentFilterType;
+  value: string;
+}
+
+const FilterTag: React.FC<FilterTagProps> = ({
+  onDelete,
+  text,
+  type,
+  value,
+}) => {
+  const { t } = useTranslation();
+  const deleteFilter = () => {
+    onDelete({ type, value });
+  };
+
+  return (
+    <Tag
+      deleteButtonAriaLabel={t(
+        'enrolmentsPage.searchPanel.buttonRemoveFilter',
+        { name: text }
+      )}
+      onDelete={deleteFilter}
+    >
+      {text}
+    </Tag>
+  );
+};
+
+export default FilterTag;
```

Comment:
Wondering if this component should be made reusable. There are 3 of these in the codebase and only EnrolmentFilterType is different. But it is not a big deal if this is repeated